ShopperInsights/src/main/java/com/braintreepayments/api/ShopperInsightsClient.kt

```diff
@@ -76,6 +82,7 @@ class ShopperInsightsClient @VisibleForTesting internal constructor(
                 )
             )
         )
+        braintreeClient.sendAnalyticsEvent(GET_RECOMMENDED_PAYMENTS_SUCCEEDED)
```

Comment:
For other modules in v5, to make sure that we always send an analytics event with each callback, we've been creating helper methods like `callbackRecommendedPaymentsSuccess()` which would include the calling back and sending analytics (for success and failure).

Reply:
Yep, I remember the pattern from one of your PRs. I did initially write it that way, but then I took a second look and realized I was abstracting out 2 calls into a method. I concluded that the abstraction wasn't worth it at this time.